src/engine/server/lua_apidef.cpp

```diff
@@ -25,7 +26,6 @@
 
 #include <engine/server/server.h>
 
-
```

Comment:
?

Reply:
this happens when you work for about 2 days on a project and delete a line anywhere xD

Reply:
That's part of the reason why having a look at the final diff before committing is always a good idea :)